framework/src/main/java/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -142,9 +142,8 @@
     private final Elements elements;
 
     /**
-     * The set of annotations found in the stub file. Keys are simple (unqualified) names. (This may
-     * be a problem in the unlikely occurrence that a type-checker supports two annotations with the
-     * same simple name.)
+     * The set of annotations found in the stub file. Keys are names. There are two entries for each
```

Comment:
In several places you deleted `simple`, instead, could you please change it to `both fully-qualified and simple names`.  That's important information.